Documentation/spelling_wordlist.txt

```diff
@@ -285,6 +285,7 @@ committers
 conf
 confPath
 config
+configs
```

Comment:
I would squash this with the previous commit.

Reply:
Good point, otherwise it breaks the build between the two.

Alternatively, replace `configs` since it's slightly ambiguous whether it's referring to files or k8s resources. I think in practice it's fairly obvious in the context but I see no harm in being more explicit.